template.tpl

```diff
@@ -1624,15 +1623,14 @@ const setFromPath = (path, val, obj, target) => {
 };
 
 /*
- * Assumes val argument is string.
  * Interprets val according to a type typ.
  * If val is a name (referred to by ref as 'eventProperty'),
  *   interprets its value in (event)object instead.
  *
- * @param val {string} - the value to interpret
- * @param typ {string} - the type (one of number, boolean, string, default)
- * @param ref {string} - singifies whether val refers to a name in evObj
- * @param evObj {Object} - the object ref may refer to.
+ * @param {*} val - the value to interpret
+ * @param {string} typ - the type (one of number, boolean, string, default)
+ * @param {string} ref - singifies whether val refers to a name in evObj
+ * @param {Object} evObj - the object ref may refer to.
```

Comment:
Any idea why many of the existing comments have flipped position between name and type? From docs your change looks like the correct format, but the rest of the file is backwards for some reason. :confused: Did something change in a version or something?

Reply:
> Did something change in a version or something?

Not sure. Probably just my fault.